src/backend/dx11/src/lib.rs

```diff
@@ -1312,6 +1316,25 @@ impl hal::command::RawCommandBuffer<Backend> for CommandBuffer {
     }
 }
 
+bitflags! {
+    struct MemoryHeapType: u64 {
```

Comment:
semantic issue here: `XxxType` implies an enum, but you have bit flags. Should be `XxxFlags` or `XxxProperties`